src/utils/hooks/useDecodeAuthToken.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from "react";
+import { useSetRecoilState } from "recoil";
+import { dataStreamsAtom } from "src/state/dataStreams";
+import { mockDataStreamsWithRoutes } from "src/mocks/data/dataStreams";
+
+const useDecodeAuthToken = () => {
```

Comment:
Is this hook supposed to return something?

Reply:
Just a placeholder hook that will handle the auth decoding. Currently not returning anything, just setting the global state from the 'token' (mocked values).